### PR TITLE
[Model] support cogvlm2 model

### PR DESCRIPTION
**Pull request by white2018**
Before you open a pull-request, please check if a similar issue already exists or has been closed before.

### When you open a pull-request, please be sure to include the following

- [ ] A descriptive title: [xxx] XXXX
- [ ] A detailed description

Thank you for your contributions!

This PR intends to enhance the functionality of lmms-eval by supporting CogVLM2 model, which is not covered in latest version.

The code is evaluated with mmstar dataset. Here is the screeshot as follows:
image

### CODE REVIEW

File: lmms_eval/models/minimonkey.py
Comment (kcz358) [inline lmms_eval/models/minimonkey.py:1]: Hi @white2018 , is this a duplicate commit? Remember that you have add this model in the previous PR

**Review by kcz358 — COMMENTED** [review]
The cogvlm model looks good. Just wondering is the minimonkey model being added for another time?

File: lmms_eval/models/minimonkey.py
Comment (white2018) [reply lmms_eval/models/minimonkey.py:1]: Sorry! It is a wrong commit. Forget about it.

**Comment by white2018** [comment]
> The cogvlm model looks good. Just wondering is the minimonkey model being added for another time?

@kcz358 This commit about minimonkey model is just a mistake. Sorry about that.

**Comment by kcz358** [comment]
Hi @white2018 , you mind if I try rebase and drop the duplicated commits on you branch?

**Comment by white2018** [comment]
> Hi @white2018 , you mind if I try rebase and drop the duplicated commits on you branch?

OK, I try it.

**Comment by white2018** [comment]
> Thank you! @white2018 , I can also help you do it if you don't have time to do so. Since I can edit on this PR branch, I can directly push to this branch that stick to this PR in your forked repo.

That will be appreciated. Thanks a lot!

**Review by kcz358 — APPROVED** [review]

**Pull request merged**